scalameta/dialects/src/main/scala/scala/meta/dialects/package.scala

```diff
@@ -105,6 +109,7 @@ package object dialects {
     def allowOrTypes = Scala210.allowOrTypes
     def allowTraitParameters = Scala210.allowTraitParameters
     def allowLiteralTypes: Boolean = Scala210.allowLiteralTypes
+    def allowTrailingComma: Boolean = Scala210.allowLiteralTypes
```

Comment:
Haha you're totally right. I actually hate this, was trying to find something nice to say about it. Opened https://github.com/scalameta/scalameta/issues/661